pino.d.ts

```diff
@@ -80,7 +80,7 @@ interface LoggerExtras<Options = LoggerOptions> extends EventEmitter {
      * @param options: an options object that will override child logger inherited options.
      * @returns a child logger instance.
      */
-    child<ChildOptions extends pino.ChildLoggerOptions>(bindings: pino.Bindings, options?: ChildOptions): pino.Logger<Options & ChildOptions>;
+    child<ChildOptions extends pino.ChildLoggerOptions = {}>(bindings: pino.Bindings, options?: ChildOptions): pino.Logger<Options & ChildOptions>;
```

Comment:
Hi @matheusvellone, I'm wondering if `pino.ChildLoggerOptions` would have been a better default or do you have some reservations against it?